tests/e2e/testsuites/dynamically_provisioned_resize_volume_tester.go

```diff
@@ -58,44 +50,4 @@ func (t *DynamicallyProvisionedResizeVolumeTest) Run(client clientset.Interface,
 	tpod.WaitForSuccess()
 
 	defer tpod.Cleanup()
-
-}
-
-// WaitForPvToResize waiting for pvc size to be resized to desired size
-func WaitForPvToResize(c clientset.Interface, ns *v1.Namespace, pvName string, desiredSize resource.Quantity, timeout time.Duration, interval time.Duration) error {
-	By(fmt.Sprintf("Waiting up to %v for pv in namespace %q to be complete", timeout, ns.Name))
-	for start := time.Now(); time.Since(start) < timeout; time.Sleep(interval) {
-		newPv, _ := c.CoreV1().PersistentVolumes().Get(context.TODO(), pvName, metav1.GetOptions{})
-		newPvSize := newPv.Spec.Capacity["storage"]
-		if desiredSize.Equal(newPvSize) {
-			By(fmt.Sprintf("Pv size is updated to %v", newPvSize.String()))
-			return nil
-		}
-	}
-	return fmt.Errorf("Gave up after waiting %v for pv %q to complete resizing", timeout, pvName)
-}
-
-// ResizeTestPvc increases size of given persistent volume claim by `sizeIncreaseGi` Gigabytes
```

Comment:
np: `sizeIncreaseGi` -> specified. 

It would be confusing if this var name is changed in the future and a maintainer forgers to update the comment.